cli/flox/src/commands/environment.rs

```diff
@@ -2155,6 +2201,8 @@ impl Update {
         }
 
         // TODO: make this async when scraping multiple inputs
+        let span = tracing::info_span!("scrape");
+        let _guard = span.enter();
```

Comment:
shouldnt we drop (or `_guard.exit()`) this guard after the scrape?
granted, we only print the results after scraping but still.
Could also add an `instrument` to `pkgdb::scrape_input` (in addition?)

Reply:
Yes to adding a `drop(_guard)`, but I'm intentionally keeping the metrics minimal for this round of instrumentation. There's all kinds of stuff that could be useful to instrument.